relayer-cli/src/conclude.rs

```diff
@@ -207,6 +203,26 @@ impl Output {
         Output::with_success().with_msg(msg)
     }
 
+    /// Combine multiple `Output`s into a single `Output` by concatenating their results. The
+    /// combined `Output` has status `Error` if even one of the constituent `Output`s is an error.
+    pub fn combined(outputs: impl IntoIterator<Item = Output>) -> Output {
```

Comment:
I think the next step could be to try and test this PR. We can reuse the instruction from https://hermes.informal.systems/commands/upgrade/test.html but it would be good to ensure that:

1. the upgraded chain has multiple clients, which could be hosted on two other chains
2. one of the clients should be frozen, or we should have some condition to simulate the failure of upgrading at least a client, so we can make sure that the output we have here is appropriate.


I'm adding a ["needs guide update" label](https://github.com/informalsystems/ibc-rs/pulls?q=is%3Aopen+is%3Apr+label%3A%22needs+guide+update%22) to this PR to also flag that we might want to update the guide.